apps/pdc-frontend/src/styles/globals.css

```diff
@@ -195,6 +195,25 @@ p:has(a[class="utrecht-skip-link utrecht-skip-link--visible-on-focus"]) {
   padding-inline-end: var(--utrecht-navigation-toggle-button-padding-end);
 }
 
+/* openforms datalist */
+.openforms-card {
+  --of-card-padding-block-end: 0;
+  --of-card-padding-block-start: 0;
+  --of-card-padding-inline-end: 0;
+  --of-card-padding-inline-start: 0;
+}
```

Comment:
Ik ben bang dat ook diverse andere "cards in cards" dan geen padding hebben, ik denk dat je de selector specifieker moet maken: `.openforms__app-body-ofzo > .openforms-card`.

Maar dan gaat het alsnog mis, omdat die custom properties inheriten. Daarom:

```css
.openforms__app-body-ofzo > .openforms-card {
  padding: 0;
}
```